### PR TITLE
Avocado TAP output

### PR DESCRIPTION
**Pull request by jscotka**
experimental own translator from json format to TAP to stdout.

### CODE REVIEW

File: test/avocado/run-tests [hunk 1]
```diff
@@ -53,6 +53,30 @@ def prepare_avocado_tests(machine):
...
+        test_log = t['logfile']
+        test_time = t['time']
+        print "# ----------------------------------------------------------------------"
+        print "# %s (time %d secs)" % (test_name, test_time)
+        print "#"
+        if test_status == 'PASS':
+            print "ok %s %s (time %d secs)" % (counter, test_name, test_time)
+        else:
+            print "not ok %s %s (time %d secs)" % (counter, test_name, test_time)
+        print machine.execute(command="cat " + test_log, quiet=True)
```
Comment (dperpeet) [inline test/avocado/run-tests]: In our other tests we have the logs first, then the status line at the end: Move this line before https://github.com/cockpit-project/cockpit/pull/3553/files#diff-8a4ec6892b625fef5add43f5219370baR69.

**Comment by dperpeet** [comment]
With the current proposal, the results are converted to TAP after they're finished. While I think this is a good start, the question is whether we want to stream them like our other results (https://github.com/cockpit-project/cockpit/blob/0d2769816a8dfd871e3d50b28ca19db680ea3766/test/testlib.py#L767).
If you say looking at the finished results in the "nicer" form is good enough for now, we don't need to implement the streaming variant.
What do you think?

**Comment by jscotka** [comment]
Hi,
exactly, it is more less about nicer output HTML form. This was easiest way how to do it, and also is shows debug info for each test also with tracebacks.
raw log of tests is okay, but full of info, and this shows nice page.

File: test/avocado/run-tests (hunk 1 above)
Comment (jscotka) [inline test/avocado/run-tests]: I can move it before result output.
I was not sure when I've read TAP specification and I understood it that it should be after. I'll move it before.

**Comment by dperpeet** [comment]
@jscotka great job on this, I didn't think we could make it work with our current log.html so easily!

**Pull request closed without merging**